web/src/js/components/FlowView/Nav.jsx

```diff
@@ -2,14 +2,15 @@ import React  from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import classnames from 'classnames'
+import _ from 'lodash'
```

Comment:
Without this statement, it works fine in the the browser, but it will fail the tests. Throwing `ReferenceError: _ is not defined`. I guess it is because `lodash` has been import by other part of our js code, but here, when we doing unit tests, it just fails to find `_`.